docs/volume/export-volume.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 4
+sidebar_label: Export a Volume to Image
+title: ""
+keywords:
+- Volume
+Description: Export volume to image from the Volume page.
+---
+
+# Export a Volume to Image
+
+After creating a volume, you can export the volume to image by the following steps:
+
+1. Clicking the `⋮` button and selecting the `Export Image` button.
+1. Select the `Namespace` that you want to export to.
```

Comment:
I fully don't understand this line. Are we exporting the volume to an image or to an image?

Reply:
Export the volume to an image

Fixed by:
1. Select the `Namespace` of the new image.

Reply:
Then we should mention that explicitly, please.

Reply:
Thanks. Fixed in the latest commit: https://github.com/harvester/docs/commit/8e5f1c959fe6ebaa6470874a38325d884e79f64b
Maybe the browser has a cache?  Need refresh？
or the `Select the Namespace of the new image` is not clear enough?